CMakeLists.txt

```diff
@@ -338,6 +413,51 @@ else()
   target_compile_definitions(preciceCore PUBLIC PRECICE_NO_PETSC)
 endif()
 
+# Setup Ginkgo
+if (PRECICE_GinkgoParallelization)
+  # Next we configure the common kernels for the matrix assembly
+  # This CMake part is still a shortcoming from the experimental Ginkgo we are using right now
+  # TODO: Remove as soon as GPU feature is in main branch
+  set(Ginkgo_Path "${Ginkgo_DIR}/.." CACHE STRING "Path to the experimental Ginkgo Repository of branch a195f856e.")
+  include(${Ginkgo_Path}/cmake/unified_kernels.cmake)
+
+  # This command registers Ginkgo kernels for different execution backends
+  ginkgo_add_unified_kernels(preciceCore src/mapping/ginkgo_kernels/GinkgoRBFKernels.cpp)
```

Comment:
As a remember: This is still a tricky command as it adds various dependencies directly to the preciceCore target and relies on global includes and definitions, which will be a pain to maintain.

If the script ends up in the next release unchanged, then we need to either hardcode the kernel handling or copy it to our code-base and patch it.